.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -96,16 +106,16 @@ jobs:
       contents: 'read'
       id-token: 'write'
     outputs:
-      custom-version-json: ${{ inputs.build-branch && steps.render-rawls-version.outputs.custom-version-json || ''}}
+      custom-version-json: ${{ needs.init-github-context.outputs.build-branch == 'true' && steps.render-rawls-version.outputs.custom-version-json || ''}}
```

Comment:
Custom version building:

<img width="510" alt="image" src="https://github.com/broadinstitute/rawls/assets/484484/5475592c-3385-4004-98a9-0845eb55fe52">

<img width="643" alt="image" src="https://github.com/broadinstitute/rawls/assets/484484/b5b19418-cb1c-4b9c-92cd-db9f1a9d26fb">

vs. not

<img width="561" alt="image" src="https://github.com/broadinstitute/rawls/assets/484484/7530b77e-8020-4550-8aa2-aa72202ef582">
<img width="439" alt="image" src="https://github.com/broadinstitute/rawls/assets/484484/198b3c66-e8df-4dea-b720-48afa16deda6">